cmd/juju/controller/addmodel.go

```diff
@@ -396,6 +398,16 @@ there is no default cloud defined, please specify one using:
 	return cloudTag, cloud, nil
 }
 
+var ambiguousDetectedCredentialError = errors.New(`
+more than one credential detected
```

Comment:
The names are available when you run "autoload-credentials". Since you can't select a detected credential by name here, I don't see the point in telling the user ahead of them running autoload-credentials.